apps/nextjs-website/src/lib/fetch/types/PageSection.ts

```diff
@@ -296,6 +296,17 @@ const ServiceCarouselSectionCodec = t.strict({
   cards: t.array(ServiceCardCodec),
 });
 
+const HighlightBoxSectionCodec = t.strict({
+  __component: t.literal('sections.highlight-box'),
+  eyelet: t.union([t.string, t.null]),
+  title: t.string,
+  body: t.string,
+  buttonText: t.union([t.string, t.null]),
+  buttonHref: t.union([t.string, t.null]),
+  imageUrl: StrapiImageRequiredSchema,
```

Comment:
```suggestion
  image: StrapiImageRequiredSchema,
```
Calling the whole image object "imageUrl" is confusing